doc/src/try-plrust-with-docker.md

```diff
@@ -0,0 +1,74 @@
+# Try PL/Rust with Docker
+
+If you would like to give PL/Rust a try without succumbing to the long arduous process of installing everything that is required, then this next section is for you!
```

Comment:
Lets maybe tone this down a bit.  Yeah, plrust is complex to install, but we don't need to talk it down.  :)